docs/radicle.md

```diff
@@ -4,13 +4,13 @@ title: What is Radicle?
 sidebar_label: What is Radicle?
 ---
 
-Radicle is an open-source stack for code collaboration 🌱. It's a peer-to-peer network built on Git that gives developers a way to collaborate on code without relying on intermediaries. It's a robust, functional, and secure alternative to centralized code collaboration platforms.
+Radicle is an open-source stack for code collaboration 🌱. Built on Git, it enables developers to collaborate on code without relying on intermediaries. It's a robust, functional, and secure alternative to centralized code collaboration platforms.
```

Comment:
```suggestion
Radicle is an open-source stack for code collaboration 🌱. It enables developers to collaborate on code without relying on intermediaries. It's a robust, functional, and secure alternative to centralized code collaboration platforms.
```

It might be misleading to use the implication that when something is built on Git it enables intermediate free usage, Github is built on Git.